app/src/main/java/uy/com/bix/app/smsproject/classes/SmsReceiver.java

```diff
@@ -0,0 +1,48 @@
+package uy.com.bix.app.smsproject.classes;
+
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.telephony.SmsMessage;
+import android.util.Log;
+import android.widget.Toast;
+
+public class SmsReceiver extends BroadcastReceiver {
+	private String TAG = SmsReceiver.class.getSimpleName();
+
+	public SmsReceiver() {
+	}
+
+	@Override
+	public void onReceive(Context context, Intent intent) {
+		// Get the data (SMS data) bound to intent
+		Bundle bundle = intent.getExtras();
+
+		SmsMessage[] msgs = null;
+
+		String str = "";
```

Comment:
@alebarreiro great comment, thanks! That's what happens when you steal some code from the web